tutoroars/templates/oars/jobs/init/configure-oars-lms.sh

```diff
@@ -0,0 +1,19 @@
+cat > /tmp/erb_config.json <<EOF
+{
+  "model": "event_routing_backends.RouterConfiguration",
+  "data":
+  [
+    {
+      "enabled": true,
+      "backend_name": "xAPI",
+      "route_url": "http://ralph:8100/xAPI/",
+      "auth_scheme": "Basic",
+      "username": "{{ RALPH_LMS_USERNAME }}",
+      "password": "{{ RALPH_LMS_PASSWORD }}"
+    }
+  ]
+}
+EOF
+
+./manage.py lms manage_user tutor-contrib-oars oars@tcril --unusable-password
+./manage.py lms populate_model -f /tmp/erb_config.json -u tutor-contrib-oars
```

Comment:
I'm getting an error on this init step. I think it's because the `OPENEDX_EXTRA_PIP_REQUIREMENTS` aren't installed on the lms _job_, though they're installed on the LMS main container? I'm not sure how best to address this.

```
Importing JSON data from file /tmp/erb_config.json
Traceback (most recent call last):
  File "/openedx/venv/lib/python3.8/site-packages/django/apps/registry.py", line 156, in get_app_config
    return self.app_configs[app_label]
KeyError: 'event_routing_backends'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./manage.py", line 106, in <module>
    execute_from_command_line([sys.argv[0]] + django_args)
  File "/openedx/venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/openedx/venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/openedx/venv/lib/python3.8/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/openedx/venv/lib/python3.8/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/openedx/venv/lib/python3.8/site-packages/config_models/management/commands/populate_model.py", line 76, in handle
    created_entries = deserialize_json(data, options['username'])
  File "/openedx/venv/lib/python3.8/site-packages/config_models/utils.py", line 61, in deserialize_json
    serializer_class = get_serializer_class(apps.get_model(parsed_json["model"]))
  File "/openedx/venv/lib/python3.8/site-packages/django/apps/registry.py", line 206, in get_model
    app_config = self.get_app_config(app_label)
  File "/openedx/venv/lib/python3.8/site-packages/django/apps/registry.py", line 163, in get_app_config
    raise LookupError(message)
LookupError: No installed app with label 'event_routing_backends'.
ERROR: 1
Error: Command failed with status 1: docker-compose -f /data/jill/.tutor/env/local/docker-compose.yml -f /data/jill/.tutor/env/dev/docker-compose.yml -f /data/jill/.tutor/env/dev/docker-compose.tmp.yml --project-name tutor_dev -f /data/jill/.tutor/env/local/docker-compose.jobs.yml -f /data/jill/.tutor/env/dev/docker-compose.jobs.yml -f /data/jill/.tutor/env/dev/docker-compose.jobs.tmp.yml run --rm lms-job sh -e -c cat > /tmp/erb_config.json <<EOF
{
  "model": "event_routing_backends.RouterConfiguration",
  "data":
  [
    {
      "enabled": true,
      "backend_name": "xAPI",
      "route_url": "http://ralph:8100/xAPI/",
      "auth_scheme": "Basic",
      "username": "lms",
      "password": "bkjnytjnslmauhasvfjwytrumkgitzvmjsln"
    }
  ]
}
EOF

./manage.py lms manage_user tutor-contrib-oars oars@tcril --unusable-password
./manage.py lms populate_model -f /tmp/erb_config.json -u tutor-contrib-oars
```

FYI when I run these commands in the lms shell (`tutor dev dc exec lms bash`), I get this error:

```
Importing JSON data from file /tmp/erb_config.json
Traceback (most recent call last):
  File "/openedx/venv/lib/python3.8/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/openedx/venv/lib/python3.8/site-packages/django/db/backends/mysql/base.py", line 73, in execute
    return self.cursor.execute(query, args)
  File "/openedx/venv/lib/python3.8/site-packages/MySQLdb/cursors.py", line 206, in execute
    res = self._query(query)
  File "/openedx/venv/lib/python3.8/site-packages/MySQLdb/cursors.py", line 319, in _query
    db.query(q)
  File "/openedx/venv/lib/python3.8/site-packages/MySQLdb/connections.py", line 254, in query
    _mysql.connection.query(self, query)
MySQLdb.ProgrammingError: (1146, "Table 'openedx.event_routing_backends_routerconfiguration' doesn't exist")
```

To get around that, I have to run migrations first:

```
./manage.py lms migrate
```

And then it completes fine.

Reply:
I think this one was related to this, which we ran into yesterday and should be fixed as of last night: https://github.com/overhangio/tutor/pull/814#event-8740794860

Reply:
Oo interesting.. huzzah for being on the cutting edge 😁

Reply:
Yep! I can confirm that this is fixed, thank you @kdmccormick !

Reply:
No problem!

Reply:
I'll give this a shot, but I don't think it's related to this code so I'm going to merge in spite of it.

Reply:
Huh, is it possible that the edx-event-routing-backends package is installed into LMS & CMS, but that the event_routing_backends app is just missing from INSTALLED_APPS?

If the package is installed, running `pip freeze | grep edx-event-routing-backends` in the LMS/CMS containers would print its version.

Reply:
Just checked this and indeed it's not installed. I believe e-r-b uses the newer style plugin system so shouldn't need to be manually added to the installed apps.

Reply:
> Just checked this and indeed it's not installed.

Yep, me neither.

> I believe e-r-b uses the newer style plugin system so shouldn't need to be manually added to the installed apps.

Confirmed, see [EventRoutingBackendsConfig](https://github.com/openedx/event-routing-backends/blob/master/event_routing_backends/apps.py#L9-L30).

Also, when I manually installed `edx-event-routing-backends` on the LMS (and CMS) containers, then the init scripts worked. 

Interestingly, [tutor's default value](https://github.com/overhangio/tutor/blob/c788bd499f75f9ef32effdd9240d22dcbd371fc6/tutor/templates/config/defaults.yml#L55-L56) is:
```OPENEDX_EXTRA_PIP_REQUIREMENTS:
  - "openedx-scorm-xblock>=15.0.0,<16.0.0"
```
But I don't see the scorm xblock in my pip freeze either?

Could something be preventing `OPENEDX_EXTRA_PIP_REQUIREMENTS` from being used on `tutor local`?